docs/source/tracking.rst

```diff
@@ -156,6 +161,11 @@ a Postgres database for backend entity storage, and an S3 bucket for artifact st
 
 .. figure:: _static/images/scenario_4.png
 
+.. code-block:: bash
+    :caption: Command to run the tracking server in this configuration
+
+    mlflow server --backend-store-uri postgresql:://URI --default-artifact-root s3://bucket_name --host remote_host --no-serve-artifacts
```

Comment:
I think we should remove the double colons in these examples, right?

```suggestion
    mlflow server --backend-store-uri postgresql://URI --default-artifact-root s3://bucket_name --host remote_host --no-serve-artifacts
```

Reply:
Can we also replace `URI` with a realistic example?

Reply:
Done!